tools/prepare-machine.ps1

```diff
@@ -122,10 +122,10 @@ function Download-eBpf-Nuget {
         [string]$Platform
     )
     # Download private eBPF Nuget package.
-    $EbpfNugetVersion = "eBPF-for-Windows.$Platform.0.20.0"
-    $EbpfNugetBuild = ""
-    $EbpfNuget = "$EbpfNugetVersion$EbpfNugetBuild.nupkg"
-    $EbpfNugetUrl = "https://github.com/microsoft/xdp-for-windows/releases/download/main-prerelease/$EbpfNugetVersion$EbpfNugetBuild.nupkg"
+    $EbpfVersion = Get-EbpfVersion
```

Comment:
this function should no longer be necessary, since the nuget will be restored from nuget.org. i would like to take this change to fix the function (history shows it comes in handy very often) but remove the callers in the `$ForBuild` case